evap/staff/templates/staff_user_list.html

```diff
@@ -0,0 +1,105 @@
+{% extends 'staff_base.html' %}
+
+{% load static %}
+
+{% block breadcrumb %}
+    {{ block.super }}
+    <li class="breadcrumb-item"><a href="{% url 'staff:user_index' %}">{% trans 'Users' %}</a></li>
+    <li class="breadcrumb-item">{% trans 'User list' %}</li>
+{% endblock %}
+
+{% block content %}
+    {{ block.super }}
+    <div class="row mb-3 align-items-center">
+        <div class="col-auto">
+            <div class="btn-switch btn-switch-light">
+                <div class="btn-switch-label">{% trans 'Inactive users' %}</div>
+                <div class="btn-switch btn-group">
+                    <a href="{% url 'staff:user_list' %}?filter_users=false" role="button" class="btn btn-sm btn-light{% if not filter_users %} active{% endif %}">
+                        {% trans 'Show' %}
+                    </a>
+                    <a href="{% url 'staff:user_list' %}?filter_users=true" role="button" class="btn btn-sm btn-light{% if filter_users %} active{% endif %}">
+                        {% trans 'Hide' %}
+                    </a>
+                </div>
+            </div>
+        </div>
+        <div class="input-group col-3">
+            <input type="search" name="search" class="form-control" placeholder="{% trans 'Search...' %}" />
```

Comment:
the search field is currently not working. probably the TableGrid initialization below broke during moving the list to this new page.